<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OLMainWindow</class>
 <widget class="QMainWindow" name="OLMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SweetPicker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="hauptgridLayout">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QPushButton" name="milkyWayButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="OLMainWindow.qrc">
           <normaloff>:/Objects/GUIMilkyWay.png</normaloff>:/Objects/GUIMilkyWay.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>220</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="snickersButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="OLMainWindow.qrc">
           <normaloff>:/Objects/GUISnickers.png</normaloff>:/Objects/GUISnickers.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>220</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="bountyButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="OLMainWindow.qrc">
           <normaloff>:/Objects/GUIBounty.png</normaloff>:/Objects/GUIBounty.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>220</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="twixButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="OLMainWindow.qrc">
           <normaloff>:/Objects/GUITwix.png</normaloff>:/Objects/GUITwix.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>220</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="doveButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="OLMainWindow.qrc">
           <normaloff>:/Objects/GUIDove.png</normaloff>:/Objects/GUIDove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>220</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="teasersButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="OLMainWindow.qrc">
           <normaloff>:/Objects/GUITeasers.png</normaloff>:/Objects/GUITeasers.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>220</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="2">
        <widget class="QLabel" name="nachrichtenprotokollLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Nachrichtenprotokoll</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="livebildLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>75</horstretch>
           <verstretch>75</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="livetextLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Livebild</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="referenztextLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Referenzbild</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="referenzbildLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>75</horstretch>
           <verstretch>75</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QTextBrowser" name="NachrichtentextBrowser"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1277</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OLMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
